platform/os_linux.go

```diff
@@ -115,12 +128,15 @@ func SetOutboundSNAT(subnet string) error {
 
 // ClearNetworkConfiguration clears the azure-vnet.json contents.
 // This will be called only when reboot is detected - This is windows specific
-func ClearNetworkConfiguration() (bool, error) {
+func (p *execClient) ClearNetworkConfiguration() (bool, error) {
 	return false, nil
 }
 
-func KillProcessByName(processName string) error {
-	p := NewExecClient()
+func (p *execClient) ExecutePowershellCommand(_ string) (string, error) {
```

Comment:
@tamilmani1989 This is a bit odd as this is linux file. Would be nice to change the name altogether to `ExecShellCommand` or something.